katas/content/distinguishing_unitaries/y_xz/Solution.qs

```diff
@@ -0,0 +1,14 @@
+namespace Kata {
+    operation DistinguishYfromXZ (unitary : (Qubit => Unit is Adj+Ctl)) : Int {
+        use qs = Qubit[2];
+        // prep (|0⟩ + |1⟩) ⊗ |0⟩
+        within { H(qs[0]); }
```

Comment:
Probably worth rewriting in the same style as the previous two tasks, with one H and MResetX